src/components/elements/background-image.tsx

```diff
@@ -20,6 +20,7 @@ interface BackgroundProps extends IBackgroundImageProps {
     data: ImageDataLike | IGatsbyImageData
     is_unstyled?: boolean
     style?: CSSProperties
+    loading?: string
```

Comment:
May I know what is the `loading` for and why it is a `string`?

Reply:
<img width="692" alt="Screenshot 2022-03-29 at 11 58 22 AM" src="https://user-images.githubusercontent.com/81704069/160597294-8319eacf-eadf-4037-b219-7888f4e003fe.png">

this is where they're using it and typescript is throwing error, so I just added it.

Reply:
I'm not sure if this `loading` prop is doing anything, because on the `gatsby-background-image` itself there's no `loading` prop declared

```
 interface IBackgroundImageProps {
    fixed?: IFixedObject | IFixedObject[] | (IFixedObject | string)[];
    fluid?: IFluidObject | IFluidObject[] | (IFluidObject | string)[];
    fadeIn?: string | boolean;
    durationFadeIn?: number;
    title?: string;
    alt?: string;
    critical?: boolean;
    crossOrigin?: string | boolean;
    backgroundColor?: string | boolean;
    onLoad?: () => void;
    onStartLoad?: (param: { wasCached: boolean }) => void;
    onError?: (event: any) => void;
    preserveStackingContext?: boolean;
    rootMargin?: string;
    keepStatic?: boolean;
  }
  ```
  
  Can you check if removing it will make a difference? It might be a props from the old component it used to be.

Reply:
<img width="1064" alt="Screenshot 2022-03-29 at 12 26 34 PM" src="https://user-images.githubusercontent.com/81704069/160601238-d2a6ba6e-50fc-49fb-b9ee-7d7a275ba7ce.png">

this is the error from typescript

Reply:
Try removing the `loading` props here I don't think it's making a difference here. But observe how it is rendering this image.

Reply:
in the home folder the file

Reply:
it has nothing to the with images, I just saw the error and tried to fixed it